python/cherrypy/config.yaml

```diff
@@ -0,0 +1,11 @@
+framework:
+  github: cherrypy/cherrypy
+  version: 18.3.0
+
+command: >
```

Comment:
you could you a non-blocking worker ? `uvloop`, `gevent`, `meinheld`

Reply:
what is the purpose of this?

Reply:
has a as-fair-as we can python benchmark, **wsgi** use `meinheld` and `asgi` use `uvicorn` (or else)

Reply:
well, CherryPy can do WSGI on both server and app sides

Reply:
fyi: I can measure an average of **169.65 ms** with `meinheld`, **233.19 ms** without any specific `gunicorn` **worker** and **166.45 ms** for `uwsgi`

Reply:
@waghanza the doc only shows a possible use of `uwsgi`, as I said earlier, there's no "recommended" way. And it's only listed in docs because somebody contributed it there.